tagstudio/src/qt/main_window.py

```diff
@@ -181,15 +212,24 @@ def setupUi(self, MainWindow):
 	def retranslateUi(self, MainWindow):
 		MainWindow.setWindowTitle(QCoreApplication.translate(
 			"MainWindow", u"MainWindow", None))
+        # Navigation buttons
 		self.backButton.setText(
 			QCoreApplication.translate("MainWindow", u"<", None))
 		self.forwardButton.setText(
 			QCoreApplication.translate("MainWindow", u">", None))
+  
+        # Search field
 		self.searchField.setPlaceholderText(
 			QCoreApplication.translate("MainWindow", u"Search Entries", None))
 		self.searchButton.setText(
 			QCoreApplication.translate("MainWindow", u"Search", None))
+  
+        # Search type selector
+		self.comboBox_2.setItemText(0, QCoreApplication.translate("MainWindow", "And (includes all tags)"))
+		self.comboBox_2.setItemText(1, QCoreApplication.translate("MainWindow", "Or (includes any tag)"))
```

Comment:
Use "Title Case" for the combo box options
```suggestion
		self.comboBox_2.setItemText(0, QCoreApplication.translate("MainWindow", "And (Includes All Tags)"))
		self.comboBox_2.setItemText(1, QCoreApplication.translate("MainWindow", "Or (Includes Any Tag)"))
```